Python_prog/addition.py

```diff
@@ -0,0 +1,5 @@
+# program to take two number as input and print sum of it
+num1=int(input("Enter a number1: "))
+num2=(int(input("Enter number2: ")))
+sum=num1 + num2
+print(f"{num1}+{num2}={sum}")
```

Comment:
use spaces properly to format output properly

Reply:
sure sir